### ISSUE TITLE
privacy issue

### ISSUE THREAD

**Issue by wouter76** [Xtraball/Siberian#423]
Hi,

Last week my server, on which Siberian CMS PE platform is hosted, is audited for security/privacy risks. I received some suggestions to comply to the Dutch privacy laws. I share the suggestions I received below.

1. passwords are not seeded yet, as this should be a design by default thing for over 5 years now;
2. personal data is not hashed when stored in a database while this is strongly recommended;
3. no log is beeing kept by who (which user) is accessing and/or updating personal data (most important user data and metadata);
4. no two factor authorization active for the app management website, which is suggested to implement for a second layer of protection of the user data.

So basically all suggestions are about the protection of the user data and procedures to be sure they are kept safe in case of a hack. 

I know you guys do a lot of work to improve the Siberian CMS platform which is great. I hope you will consider, at least the first two, suggestions I received.   

Kind regards,
Wouter

**Comment by userdevel** [comment]
@Slicit, @LikoV2: Any news after more one year?
